### ISSUE TITLE
TypeScript Support

### ISSUE THREAD

**Issue by buffet-time** [prantlf/rollup-plugin-swc-minify#1]
Hello! 

Would you mind me submitting a PR to migrate this plugin into TypeScript? I currently use:
```ts
// @ts-expect-error
import { minify } from 'rollup-plugin-swc-minify'
...
// eslint-disable-next-line @typescript-eslint/no-unsafe-call
plugins: [minify()]
```

So typescript doesn't yell at me, but would be nice not to have to have these.

**Comment by prantlf** [comment]
Yes, of course, don't be shy, you can just create a PR any time ;-) It'll be a welcome addition.
You can have a look at the typings in [rollup-plugin-jsonlint](https://github.com/prantlf/rollup-plugin-jsonlint). It means adding a `.d.ts` file, exposing it in `package.json` and optionally a simple unit tests for consuming it by `tsc`. Thank you!
